types/module/core_module.go

```diff
@@ -122,9 +122,15 @@ func (c coreAppModuleBasicAdaptor) InitGenesis(ctx sdk.Context, cdc codec.JSONCo
 	}
 
 	if mod, ok := c.module.(HasGenesis); ok {
-		return mod.InitGenesis(ctx, cdc, bz)
+		mod.InitGenesis(ctx, cdc, bz)
```

Comment:
The return should still be there

Reply:
nothing to return here though, not changing the name of the other one will add it for that one though